forwards_paper.tex

```diff
@@ -822,46 +822,42 @@ \subsubsection*{Computational complexity}
 but, how does it perform in theory?
 % The simulation results shown in Figures \ref{fig:runtimes_selection} and \ref{fig:relative_speedup_selection} show that
 % pedigree tracking greatly speeds up forward-time simulations.  Further, the run times with pedigree tracking become
-% near-linear in $\rho$ (Figure \ref{fig:runtimes_selection}).  
-In this section, we explore the how required computational effort and storage space scale
+% near-linear in $\rho$ (Figure \ref{fig:runtimes_selection}).
+In this section, we explore how the required computational effort and storage space scale
 in a Wright-Fisher model with recombination.
 
 %% Memory reduction thanks to simplification
-Consider a simulation of a Wright-Fisher population of $2N$ haploid individuals
+Consider a simulation of a Wright-Fisher population of $N$ haploid individuals
 using Algorithm~\algref{W} for $T$ generations,
 with simplify interval $s = 1$ so that redundant information is removed after every generation.
 Since every chromosome inherits material from both parents,
 without simplification this would produce tables of
-$2NT$ nodes and
-$4NT$ edges.
-Suppose that $T$ is large enough that all samples coalesce within the simulation with high probability,
-($T = 20N$, say).
+$NT$ nodes and
+$2NT$ edges.
+Suppose that $T$ is large enough that all samples coalesce within the simulation with high probability
+($T = 20N$, say, giving a total space requirement of $60N^2$).
 After simplification, we are left with the tree sequence describing the history
-of only the current generation of $2N$ individuals.
-If no coalescent events involve more than two lineages, this tree sequence has $4N-2$ edges to describe the leftmost tree;
+of only the current generation of $N$ individuals.
```

Comment:
I think this was wrong. We have 2N - 1 nodes and 2N - 2 edges in the first tree, right? So, if we had N==2N, this would be 4N - 4.

Reply:
A rooted tree with `k` tips has `2k-2` edges (since: with `k=2` there are 2 edges; adding a tip adds 2 edges), so it was right before, but it is still right, so all is good. =)

Reply:
A rooted tree of `2N` tips has `4N-2` edges (check for `N=1`; adding each tip adds two edges), so I think it was right before?  but, still right, so all is good...

Reply:
I've got:
```
N    edges
1     0
2     2
3     4
4     6
```
which is 2N - 2. Therefore, if we have N=2K, then this is 4K - 4, right? Or am I missing something about the 2N?

Reply:
I am confused?   Why is it not
```
2N-2 = 2(2K)-2 = 4K-2
```

Reply:
Yes. Doh. I'm not used to this 2N stuff!

Reply:
it is tradition in popgen, to remind ourselves that organisms are diploid... ?